browser/about/acofone/ac-of-one-part-one.md

```diff
@@ -0,0 +1,13 @@
+## Executive Summary
+
+The vast majority of variants being discovered within large population datasets – and in particular those with the greatest functional impact – are extremely rare. Some genomic data generating programs limit the ability to share allele counts (AC) below a certain threshold (e.g. below 20 in the All of Us Research Program unless an exception is granted). However, sharing summary statistics for all variants, including those found only in a single individual, is critical for scientific progress and discovery. Obscuring information about this rare variation would be reasonable if the risks to participants were large, but the experiences of many other large genomics initiatives illustrate that the actual risks to privacy are quite small.
+
+<br />
```

Comment:
Whither the `br`s here?

Reply:
They're just adding space to make the page more readable, is it better to use css (via styled components in our case) to accomplish this?